src/task/methods/scape/script.py

```diff
@@ -45,6 +48,24 @@
 # load log pvals
 df_de = scape.io.load_slogpvals(par['de_train']).drop(columns=["id", "split"], axis=1, errors="ignore")
 
+def confirm_celltype(cell, sm_name=None):
+	global df_de
```

Comment:
why not just pass `df_de` as an argument, that is:

```suggestion
def confirm_celltype(df_de, cell, sm_name=None):